.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: Static code analysis
+
+on: push
+
+jobs:
+
+  flake8:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - run: |
+          python -m pip install --upgrade pip
+          pip install flake8  flake8-import-order flake8-blind-except flake8-builtins
+
+      - run: flake8 --max-line-length=120 --import-order-style=pycharm --statistics --application-import-names api find_new harvest_products
+
```

Comment:
Multiple `--application-import-names` should be given as [a comma separated list](https://pypi.org/project/flake8-import-order/#configuration:~:text=a%20comma%20separated%20list,-of%20names%20that%20should%20be%20considered%20local%20to%20your%20application), otherwise the `[1:]` arguments are telling flake8 which packages to analyze